### PR TITLE
Fix pingserver

### PR DESCRIPTION
**Pull request by thinkingfish**
- Added error handler
- Allow 'ping' (lower case)

error handling and 'ping' individually tested (not included in the PR)

### CODE REVIEW

**Comment by thinkingfish** [comment]
@brayniac Does this look good to you?

**Comment by brayniac** [comment]
@thinkingfish yes, looks good

File: src/protocol/data/ping/parse.c [hunk 1]
```diff
@@ -46,7 +46,8 @@ parse_req(struct buf *buf)
         return PARSE_EUNFIN;
     }
 
-    if (cc_memcmp(buf->rpos, REQUEST, REQ_LEN) == 0) {
+    if (cc_memcmp(buf->rpos, REQUEST_UPPER, REQ_LEN) == 0 ||
+            cc_memcmp(buf->rpos, REQUEST_LOWER, REQ_LEN) == 0) {
```
Comment (seppo0010) [inline src/protocol/data/ping/parse.c]: why not `strncasecmp`?

Comment (thinkingfish) [inline src/protocol/data/ping/parse.c]: Uhh, because I didn't do my home work. Good call.

Comment (thinkingfish) [inline src/protocol/data/ping/parse.c]: Updated & tested again. Thanks!

**Pull request merged**